core/src/types/ethereum_structs.rs

```diff
@@ -1,2 +1,95 @@
-//! Ethereum bridge struct re-exports.
+//! Ethereum bridge struct re-exports adn types to do with ethereum.
```

Comment:
```suggestion
//! Ethereum bridge struct re-exports and types to do with ethereum.
```
just this type here. otherwise lgtm